harvard-lts/drs-migration-dashboard
# FE work from week 1

**FE work from week 1**
* * *

# What does this Pull Request do?
Contains all of the front end work that was done in week 1.

# How should this be tested?

Go to [https://localhost:3001/migrationstatus/piechart/](https://localhost:3001/migrationstatus/piechart/) to see the latest updates:

* Modal windows will pop up when you click on the "status definitions" and "status diagram" links
* Tab through each card to view the different layouts of the data (currently have the doughnut charts and data tables)
* Go to the [google spreadsheet](https://docs.google.com/spreadsheets/d/1FFv-csMLes-c6rQWCQBZa1PvrlY0b78XpQs8LpppUZA/edit#gid=0) and add a new row of data. Then refresh the page and these changes should be reflected in the date, time, doughnut charts, and data tables (note: i think there needs to be something in each cell before anything will appear)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
@phil-plencner-hl @dl-maura
